crates/project/src/project.rs

```diff
@@ -8425,28 +8437,24 @@ impl Project {
                         );
                     }
                 })
-            } else if abs_path.ends_with(local_launch_file_relative_path()) {
```

Comment:
Can you update this to use the `path` instead of `abs_path` I saw this was causing a panic on main. Some for the one aboth.

Reply:
When is it causing a panic on main?

Reply:
See the following PR: 
https://github.com/zed-industries/zed/pull/13554

Reply:
I updated my branch and fixed the issue. Thank you for letting me know!!